namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-prod/09-prometheus-sqs-sns.yaml

```diff
@@ -15,19 +15,40 @@ spec:
         message: SQS - {{ $labels.queue_name }} has message older than 15mins, check consumers are healthy.  https://grafana.live.cloud-platform.service.justice.gov.uk/d/AWSSQS000/aws-sqs?orgId=1&var-datasource=Cloudwatch&var-region=default&var-queue={{ $labels.queue_name }}&from=now-6h&to=now
         runbook_url: https://dsdmoj.atlassian.net/wiki/spaces/NOM/pages/1739325587/DPS+Runbook
       expr: |-
-        sum(aws_sqs_approximate_age_of_oldest_message_maximum{queue_name!~"Digital-Prison-Services-prod-hmpps_workload_offender_events_queue|Digital-Prison-Services-prod-hmpps_allocation_required|Digital-Prison-Services-prod-hmpps_tier_calculation_allocation|Digital-Prison-Services-prod-hmpps_allocations_offender_events_queue",queue_name!="Digital-Prison-Services-prod-prison_to_probation_update_mirror.*",queue_name!~"Digital-Prison-Services-prod-hmpps_tier.*",queue_name!="Digital-Prison-Services-prod-cfo_queue",queue_name!="Digital-Prison-Services-prod-in_cell_hmpps_queue",queue_name!="Digital-Prison-Services-prod-probation_offender_search_event_dl_queue",queue_name=~"Digital-Prison-Services-prod.*", queue_name!~".*_dl|.*dlq"} offset 5m) by (queue_name) > 15 * 60
+        sum(aws_sqs_approximate_age_of_oldest_message_maximum{
+          queue_name!~"Digital-Prison-Services-prod-hmpps_prisoner_to_nomis_queue|Digital-Prison-Services-prod-prisoner_from_nomis_queue",
+          queue_name!~"Digital-Prison-Services-prod-hmpps_workload_offender_events_queue|Digital-Prison-Services-prod-hmpps_allocation_required|Digital-Prison-Services-prod-hmpps_tier_calculation_allocation|Digital-Prison-Services-prod-hmpps_allocations_offender_events_queue",
+          queue_name!="Digital-Prison-Services-prod-prison_to_probation_update_mirror.*",
+          queue_name!~"Digital-Prison-Services-prod-hmpps_tier.*",
+          queue_name!="Digital-Prison-Services-prod-cfo_queue",
+          queue_name!="Digital-Prison-Services-prod-in_cell_hmpps_queue",
+          queue_name!="Digital-Prison-Services-prod-probation_offender_search_event_dl_queue",
+          queue_name=~"Digital-Prison-Services-prod.*",
+          queue_name!~".*_dl|.*dlq|.*_dl_queue"
+        } offset 5m) by (queue_name) > 15 * 60
```

Comment:
reformatted,
added `queue_name!~"Digital-Prison-Services-prod-hmpps_prisoner_to_nomis_queue|Digital-Prison-Services-prod-prisoner_from_nomis_queue"`,
added '.*_dl_queue' to exclusions